tests/unit/AgreementType.spec.ts

```diff
@@ -27,38 +28,41 @@ const agreementTypeTestCases = [
   {
     entityType: CorpTypeCd.BC_COMPANY,
     sampleAgreement: {
-      agreementType: 'Table-1'
+      agreementType: 'sample'
     },
     customAgreement: {
       agreementType: 'custom'
     },
-    sampleSummaryText: 'The sample Incorporation Agreement and Table 1 Articles ' +
-      'has been completed and a copy has been added to the company\'s record book.',
+    sampleSummaryText: 'The sample Incorporation Agreement and Articles ' +
+    'have been completed and a copy has been added to the company\'s record book.',
```

Comment:
Are you about has -> have?

I think the subject is "the sample XXX", which is singular.

Please confirm with UI if you haven't already.

Ditto line 39 below.

Reply:
Also fix in src/resources/Incorporations/limitedCompany.ts as needed.

Reply:
I'll double check with Yui the UI design does say have not has. I think that the sample agreement and the articles are the subject, but my grammar sucks 😢...